app/src/main/res/layout/concept_card_fragment_test_activity.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout android:layout_height="match_parent"
+  android:layout_width="match_parent"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <Button
+    android:id="@+id/open_dialog"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="Open Concept Card Dialog"
+    app:layout_constraintTop_toTopOf="parent"
```

Comment:
If this is the only child in the layout, maybe just use a FrameLayout instead. Constraints don't seem needed here.